common/djangoapps/student/views.py

```diff
@@ -118,6 +117,12 @@
 import analytics
 from eventtracking import tracker
 
+# Note that this lives in LMS, so this dependency should be refactored.
```

Comment:
Not really-- I don't know what the value is of having "TODO" (with nothing on any backlog).